cob_voltage_control/ros/src/savitzky_golay.py

```diff
@@ -74,9 +73,9 @@ def process_voltage(self):
 
         self.msg_power.header.stamp = rospy.Time.now()
         self.msg_power.time_remaining.secs = self.t_est
-        self.msg_power.prediction_method = '3rd_order_polynom'
-        self.msg_power.relative_capacity = (self.t_est/self.maximum_time) * 100
-        self.msg_power.AC_present = 0
+        #self.msg_power.prediction_method = '3rd_order_polynom'
+        #self.msg_power.relative_capacity = (self.t_est/self.maximum_time) * 100
+        #self.msg_power.AC_present = 0
```

Comment:
These fields are not in the message anymore?

Reply:
exactly